build/Dockerfile

```
@@ -1,6 +1,17 @@
-FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
+FROM registry.access.redhat.com/ubi8/ubi:8.0
```

```suggestion
FROM registry.access.redhat.com/ubi8/ubi:latest
```

The `8.0` tag is already old (there's an `8.1` already) & has a 'C' grade on the ecosystem catalog. I'd prefer `latest` than `8.1` since when it goes to `8.2` we'd probably forget to update.

Just so we don't miss something, did you copy this from another operator?

---

Im happy to change it to the latest tag. I grabbed that from the Integreatly Operator Dockerfile

---

Since you're changing it here anyway, I'd go ahead and update.

@philbrookes the integreatly-operator should probably do the same thing I think.

---

Or actually, if you want, you could just leave it the way it was @austinmartinh 

```suggestion
FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
```

The AMO is expected to run on OpenShift v3.11, which is different from the integreatly-operator or cloud-resource-operator.